src/org/jetbrains/java/decompiler/modules/decompiler/SwitchPatternMatchProcessor.java

```diff
@@ -262,6 +266,41 @@ private static boolean processStatement(SwitchStatement stat, Statement root) {
       }
     }
 
+    Exprent oldSelector = realSelector;
```

Comment:
Since this is a large amount of code, it should have a heading describing what it's trying to do. Such as:

```suggestion
    // Try to inline:
    // Objects.requireNonNull(var);
    // switch (var) {
    // 
    // to
    // switch (var) {
    
    Exprent oldSelector = realSelector;
```